apiman/src/main/java/io/fabric8/apiman/KubernetesServiceCatalog.java

```diff
@@ -97,8 +100,7 @@
 		}
 		osClient.close();
```

Comment:
`KubernetesClient` & `OpenShiftClient` implement `AutoCloseable` so might be better to use try-with-resources here to ensure client is properly closed in case of any errors. Potential resource leak?